src/ripple/app/misc/impl/AMMHelpers.cpp

```diff
@@ -165,31 +165,48 @@ adjustAmountsByLPTokens(
 
     if (lpTokensActual < lpTokens)
     {
+        bool const ammRoundingEnabled = [&]() {
+            if (auto const& rules = getCurrentTransactionRules();
+                rules && rules->enabled(fixAMMRounding))
```

Comment:
When this is merged with develop will `fixAMMRounding` be an additional amendment?  Or will you be leveraging the recently added `fixAMMv1_1`?  I'm fine either way, but it's worth being aware of the merge consequences.

Reply:
I'll leverage `fixAMMv1_1`.

Reply:
I need to merge with the other AMM PR #4983.